account/identity.go

```diff
@@ -12,20 +13,60 @@ import (
 	"golang.org/x/net/context"
 )
 
-// Identity ddenDescribes a unique Person with the ALM
+const (
+	// KeycloakIDP is the name of the main Keycloak Identity Provider
+	KeycloakIDP string = "kc"
```

Comment:
well, "kc" means the main KC instance regardless or what domain we use. For test, dev, production it can be different domains. This field is more like a type. But we could add a new field IDP-url or something which maybe usefule for remote sources. But even for github, jira, etc we would need a type of the IDP too. Otherwise how UI can distinguish JIRA from GitHub for example.